### PR TITLE
feat: proxy point && delay

### PR DESCRIPTION
**Pull request by Zephyruso**
render proxy point && delay 
image
image

### CODE REVIEW

**Review by kunish — APPROVED** [review]
Looking great

File: src/pages/Proxies.tsx [hunk 1]
```diff
@@ -5,32 +5,68 @@ import type { Proxy, ProxyProvider } from '~/types'
...
+                <input type="checkbox" />
+                <div class="collapse-title text-xl font-medium">
+                  {proxy.name}
+                </div>
+                <div class="collapse-content grid grid-cols-1 gap-2 sm:grid-cols-3  lg:grid-cols-5">
+                  <For each={proxy.all}>
+                    {(proxyPoint) => (
+                      <div
+                        class={`${
+                          proxy.now === proxyPoint
```
Comment (kunish) [inline src/pages/Proxies.tsx:60]: you can refactor this with twMerge / twJoin

**Review by kunish — CHANGES_REQUESTED** [review]

**Comment by Zephyruso** [comment]
image
refactor classname with twmerge

**Comment by kunish** [comment]
> image refactor classname with twmerge

Cool

**Review by kunish — APPROVED** [review]

**Pull request merged**